mapflow/mapflow.py

```diff
@@ -1754,6 +1754,7 @@ def download_results_callback(self, response: QNetworkReply, pid: str) -> None:
         ))
         # Add the source raster (COG) if it has been created
         raster_url = processing.raster_layer.get('tileUrl')
+        tile_json_url = raster_url.replace('/{z}/{x}/{y}.png', '.json')
```

Comment:
what if we start streaming in jpg or webp?
don't we have processing.raster_layer.get('tileJsonUrl') in response?